lib/forki/scrapers/post_scraper.rb

```diff
@@ -270,7 +270,7 @@ def parse(url)
       post_data = extract_post_data(graphql_strings)
       post_data[:url] = url
       user_url = post_data[:profile_link]
-      post_data[:screenshot_file] = save_screenshot("/tmp/#{SecureRandom.uuid}.png")
+      post_data[:screenshot_file] = save_screenshot("#{Forki.temp_storage_location}/facebook_screenshot_#{SecureRandom.uuid}.png")
```

Comment:
Same question as YouTube. Here's it's constant, previously it's determined.

Reply:
We dynamically determine the extension of downloaded media in `forki.rb`: https://github.com/TechAndCheck/forki/blob/5c100789453860ba0bf627e677e2f69b8a34e8fa/lib/forki.rb#L56-L64

But we hardcode the extension you're pointing out because the file in question comes from a screenshot, not a 3rd-party download.